tests/sentry/issues/test_escalating.py

```diff
@@ -0,0 +1,160 @@
+        group_2_id = event2.group_id
+        event3 = self._load_event_for_group(fingerprint="group-2", minutes_ago=60)
+        # Increases the count of event3
+        self._load_event_for_group(fingerprint="group-2", minutes_ago=59)
+
+        # This forces to test the iteration over the Snuba data
+        with patch("sentry.issues.escalating.QUERY_LIMIT", new=2):
+            assert query_groups_past_counts(Group.objects.all()) == [
+                {
+                    "count()": 2,
+                    "group_id": group_1_id,
+                    "hourBucket": to_start_of_hour(event1.datetime),
+                    "project_id": self.project.id,
+                },
+                {
+                    "count()": 1,
+                    "group_id": group_2_id,
+                    "hourBucket": to_start_of_hour(event2.datetime),
+                    "project_id": self.project.id,
+                },
+                {
+                    "count()": 2,
+                    "group_id": group_2_id,
+                    "hourBucket": to_start_of_hour(event3.datetime),
+                    "project_id": self.project.id,
+                },
+            ]
+
+    def test_query_multiple_projects(self) -> None:
+        proj_x = Factories.create_project(self.project.organization)
+        proj_y = Factories.create_project(self.project.organization)
+
+        event1 = self._load_event_for_group(project_id=proj_x.id)
+        # This event has the same fingerprint as event1 but
+        # should be different group IDs since they belong to different projects
+        event_y_1 = self._load_event_for_group(project_id=proj_y.id, minutes_ago=61)
+        assert event1.group_id != event_y_1.group_id
+
+        event_y_2 = self._load_event_for_group(project_id=proj_y.id, fingerprint="group-1")
+        # Increases the count of group-1
+        self._load_event_for_group(project_id=proj_y.id, fingerprint="group-1")
+
+        assert query_groups_past_counts(Group.objects.all()) == [
+            {
+                "count()": 1,
+                "group_id": event1.group_id,
+                "hourBucket": to_start_of_hour(event1.datetime),
+                "project_id": proj_x.id,
+            },
+            {
+                "count()": 1,
+                "group_id": event_y_1.group_id,
+                "hourBucket": to_start_of_hour(event_y_1.datetime),
+                "project_id": proj_y.id,
+            },
+            {
+                "count()": 2,
+                "group_id": event_y_2.group_id,
+                "hourBucket": to_start_of_hour(event_y_2.datetime),
+                "project_id": proj_y.id,
+            },
+        ]
+
+    def test_query_different_orgs(self) -> None:
+        proj_a = Factories.create_project(self.project.organization)
+        org_b = Factories.create_organization()
+        proj_b = Factories.create_project(org_b)
+
+        event1 = self._load_event_for_group(project_id=proj_a, minutes_ago=60)
```

Comment:
should this be `project_id=proj_a.id`? 

and the same for L130

Reply:
You're very right. I do not know why the tests succeeded. I will check.